gulpfile.js

```diff
@@ -94,10 +124,42 @@ gulp.task('build', function(cb) {
 			gulp.src(paths.src)
 				.pipe(gulp.dest(path.join(getBuildDir(browser), paths.dest)));
 		});
+
+		if (config[browser].manifest) {
+			del(path.join(getBuildDir(browser), config[browser].manifest));
```

Comment:
[It should](https://www.npmjs.com/package/del#del-patterns-options), we probably need to update to 2.1.0